### PR TITLE
Use sxdes

### PR DESCRIPTION
**Pull request by esheldon**
Use the sxdes package for detection.

No api changes, still requiring the sx config to be sent

### CODE REVIEW

File: volatile-requirements.txt [hunk 1]
```diff
@@ -2,3 +2,4 @@
 # a lot and usually we need master
 git+https://github.com/esheldon/ngmix.git@master
 git+https://github.com/esheldon/meds.git@master
```
Comment (beckermr) [inline volatile-requirements.txt:4]: We should pull these out of conda.

Comment (esheldon) [reply volatile-requirements.txt:4]: Yep.  I put it there so the tests could run; we can keep this PR open until sxdes gets on conda and then update this PR

Comment (beckermr) [reply volatile-requirements.txt]: this is done.

**Comment by beckermr** [comment]
@esheldon ready to merge?

**Pull request merged**